i18n/en-US.properties

```diff
@@ -432,16 +432,16 @@ be.tasks.feed.moreAssigneesLabel = See all assignees
 be.tasks.feed.rejectAction = Reject
 # Label for a rejected task
 be.tasks.feed.rejectedLabel = Task Rejected
-# accepted task status
-be.tasks.status.accepted = Accepted
-# Completed task status
-be.tasks.status.completed = Completed
+# Accepted task status, where completedAt is a human-readable time
+be.tasks.status.accepted = Accepted {completedAt}
+# Completed task status, where completedAt is a human-readable time
+be.tasks.status.completed = Completed {completedAt}
 # Task status when in progress
 be.tasks.status.inProgress = Awaiting
 # Task status when not started
 be.tasks.status.notStarted = Awaiting
-# Rejected task status
-be.tasks.status.rejected = Rejected
+# Rejected task status, where completedAt is a human-readable time
+be.tasks.status.rejected = Rejected {completedAt}
```

Comment:
Should this be `{rejectedAt}` or maybe something more generic like `{dateTime}`?